app/src/main/res/layout/repo_element.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/repo_number"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="18"
+        android:gravity="right"/>
```

Comment:
E de preferat sa folosesti end in loc de right. Ii va ajuta mult pe utilizatorii care citesc rightToLeft.